build.proj

```diff
@@ -19,7 +19,8 @@
   </Target>
 
   <Target Name="VSTest">
-    <Exec Command="dotnet test Equinox.sln $(Cfg) $(TestOptions)" />
+    <Exec Command="dotnet test tests/Equinox.MemoryStore.Integration $(Cfg) $(TestOptions)" />
```

Comment:
what's the problem with the other approach ? the cosmos branch will bring more test assemblies and I'd ideally like to have them be picked up automatically ?

Reply:
WIP testlist from cosmos (not yet pushed)

```
  <Target Name="VSTest">
    <Exec Command="dotnet test tests/Equinox.Cosmos.Integration $(Cfg) $(TestOptions)" />
    <Exec Command="dotnet test tests/Equinox.EventStore.Integration $(Cfg) $(TestOptions)" />
    <Exec Command="dotnet test tests/Equinox.MemoryStore.Integration $(Cfg) $(TestOptions)" />
    <Exec Command="dotnet test samples/Store/Integration $(Cfg) $(TestOptions)" />
    <Exec Command="dotnet test samples/Store/Domain.Tests $(Cfg) $(TestOptions)" />
  </Target>
```
ideally the above minus the cosmos one would go in this PR

Reply:
i think the issue is that dotnet test *.sln doesnt work on osx/unix, need to specify the fsproj directly (not the sln)
ln has some of quirks in os/linux according to what Enrico told me

Reply:
@michaelliao5 yes, see below - I read too fast ! But my point (is there an issue about them fixing that somewhere?) remains (for example there are 2 other test assemblies (listed above) that the sln used to pick up)